### PR TITLE
angleproject: enable d3d11

### LINKED ISSUE #14665: Header conflict between angleproject and mingw-w64-headers package
### Description / Steps to reproduce the issue

Seems mingw-w64-headers include /KHR/khrplatform.h where angleproject also includes that in that package before

### Expected behavior

file conflict resolution

### Actual behavior

File conflicts since both headers are included in each package.

### Verification

- [X] I have verified that my MSYS2 is up-to-date before submitting the report (see https://www.msys2.org/docs/updating/)

### Windows Version

MINGW64_NT-10.0-22621

### MINGW environments affected

- [X] MINGW64
- [X] MINGW32
- [X] UCRT64
- [X] CLANG64
- [X] CLANG32
- [ ] CLANGARM64

### Are you willing to submit a PR?

_No response_

### PR DESCRIPTION
**Pull request by raedrizqie**
Fixes #14665 

- enable d3d11
- removes khr header
- fix license type

### CODE REVIEW

File: mingw-w64-angleproject/PKGBUILD [hunk 1]
```diff
@@ -24,14 +24,15 @@ makedepends=("${MINGW_PACKAGE_PREFIX}-cc"
              "${MINGW_PACKAGE_PREFIX}-python"
              "git")
 _commit=("2c60b86e35652689435be5c943fbd2f7fc2d2fd7")
-options=('!buildflags')
+options=('!strip' '!buildflags')
```
Comment (lazka) [inline mingw-w64-angleproject/PKGBUILD:27]: why is this needed?

Comment (raedrizqie) [reply mingw-w64-angleproject/PKGBUILD:27]: all those exe and dll (reside in exe.unstripped & lib.unstripped dir) are stripped, before they were copied to out\Release-${MSYSTEM}

..and stripping static archives (*.a) on clang give warnings..

Comment (lazka) [reply mingw-w64-angleproject/PKGBUILD:27]: The install size increased from 37mb to 240mb though. Is that still expected?

Comment (raedrizqie) [reply mingw-w64-angleproject/PKGBUILD:27]: ah..ok.. i'll revert it back

**Comment by Biswa96** [comment]
The symbol in the added patch has been _semi-upstreamed_ in wine https://gitlab.winehq.org/wine/wine/-/commit/d059dd1cd0770e9c63da33562f6bcba4ef52846b commit. mingw-w64 will be synced probably next year's wine 8.0 release.

**Comment by raedrizqie** [comment]
May I request a library stub to be upstreamed? it is msxml2 or maybe msxml6? :)

**Comment by Biswa96** [comment]
I will absolutely do that. Can you provide any linking error output?

**Comment by raedrizqie** [comment]
its unrelated to angle package.. just want it for atl/mfc. right now i had to link it with msxml2.lib

**Comment by Biswa96** [comment]
> just want it for atl/mfc. 

How? ATL/MFC is not possible with mingw-w64 yet. Did you port it to mingw-w64? One has to implement those without looking at the actual code by Microsoft. Or the implementation has to be different like old unix vs gnu tools.

**Comment by raedrizqie** [comment]
> How? ATL/MFC is not possible with mingw-w64 yet. Did you port it to mingw-w64? 

yes..

> One has to implement those without looking at the actual code by Microsoft. Or the implementation has to be different like old unix vs gnu tools.

I'm using mfc v10 (the last version that comes with vs makefiles). Only Clang can compiles it because of `if_exists`, `if_not_exists` statements and some other things I can't remember right now..

Of course, the resulting mfc100u.dll fails at runtime though.. memory heap error or something like that..

**Comment by Biswa96** [comment]
Can libuuid.a from mingw-w64 used in place of msxml2/6 ? I see some definitions in msxml-uuid.c.

**Comment by raedrizqie** [comment]
linking with mingw's uuid and msxml2.lib just gives warnings of end directives being corrupt or sime sort.. but the linking was successfull..

**Comment by Biswa96** [comment]
Understood. Let me create msxml2/6 libraries in my fork for your testing. ⌛
